lib/components/pagination-nav.vue

```diff
@@ -90,7 +90,7 @@
             </li>
             <li v-else class="page-item" role="none presentation">
                 <b-link class="page-link"
-                        v-bind="linkProps(currentPage - 1)"
+                        v-bind="linkProps(numberOfPages)"
```

Comment:
Yeah,  I'll update the docs with a few examples, and explain that page num is a value from 1 to N, while arrays are 0 to N-1.